src/OpenColorIO/OCIOYaml.cpp

```diff
@@ -3323,8 +3338,31 @@ inline void save(YAML::Emitter& out, ConstColorSpaceRcPtr cs, unsigned int major
         }
         out << YAML::Flow << YAML::Value << aliases;
     }
+    
     out << YAML::Key << "family" << YAML::Value << cs->getFamily();
     out << YAML::Key << "equalitygroup" << YAML::Value << cs->getEqualityGroup();
+
+    const std::string interopID{cs->getInteropID()};
+    if (!interopID.empty()) 
+    {
+        out << YAML::Key << "interopid";
+        out << YAML::Value << interopID;
+    }
+
+    const std::string amfTransformIDs{cs->getAmfTransformIDs()};
+    if (!amfTransformIDs.empty()) 
+    {
+        out << YAML::Key << "amftransformids";
+        out << YAML::Value << amfTransformIDs;
+    }
+
+    const std::string iccProfileName{cs->getIccProfileName()};
+    if (!iccProfileName.empty()) 
+    {
+        out << YAML::Key << "iccprofilename";
+        out << YAML::Value << iccProfileName;
+    }
+
```

Comment:
The interop ID should go after the aliases.
The AMF IDs should go after the description.
The ICC profile should go after the AMF IDs.